### ISSUE TITLE
Steam link mobile / on screen controls

### ISSUE THREAD

**Issue by mixalis1987** [ValveSoftware/steam-for-linux#7703]
Not sure where to post this but I hope this is the right place.

When connecting to linux from phone with steam link the on screen controls do not work when the game starts ( works in steam big picture mode ) this is for all games proton and native games via steam link.

The controls work when connecting to Windows 10 

To note the touch screen moves the mouse and click function works.

**Comment by Chorwacjen** [comment]
Yeah it is still a problem, could somebody just look into this? I have tried a bit of stuff and nothing worked out, in the end you can use phone keyboard to control but this is NOT playable, and also the fact that it covers half the screen is also a problem id say. The screen controls do not work in games, even if they do in big picture mode.
